JourneysSquad/JourneysSquad/ViewModel/DummySectionViewModel.swift

```diff
@@ -0,0 +1,23 @@
+import Foundation
+
+/// Dummy view model that provides data to display for a bills list section.
+final class DummySectionViewModel: Identifiable {
+    /// The section's unique identifier.
+    var id: String { header }
+
+    /// A text shown as a header of the section in the view.
+    let header: String
+
+    /// Dummy data that provides a list of bills under the header of the section.
+    let rows: [DummyBillRowViewModel]
+
+    /// Initializes a new instance of the `SectionViewModel`.
+    ///
+    /// - Parameters:
+    ///   - header: A header text of the section.
+    ///   - rows: A list of hardcoded bills those are grouped into the current section.
```

Comment:
nit:
Why do you like word "hardcoded" ? :)
You can use "fixed" for example:
```suggestion
    ///   - rows: A list of fixed bills those are grouped into the current section.
```

Reply:
There's even better word "static": 
```suggestion
    ///   - rows: A static list of bills to show in the section.
```

Reply:
fixed. Used variant with `static list`